content/02.introduction.md

```diff
@@ -1,3 +1,16 @@
 ## Introduction
+Pediatric brain and spinal cord tumors are the second most common tumors in children (after leukemia), yet they remain a leading cause of death by disease in children @doi:10.1093/neuonc/noz150. 
```

Comment:
```suggestion

Pediatric brain and spinal cord tumors are the second most common tumors in children after leukemia, yet they remain a leading cause of death by disease in children [@doi:10.1093/neuonc/noz150]. 
```

Reply:
ahh, I thought I was missing something with these dois, but my brain wasn't figuring it out. thanks!